modules/andes-core/broker/src/main/java/org/dna/mqtt/moquette/messaging/spi/impl/ProtocolProcessor.java

```diff
@@ -594,7 +594,7 @@ void processUnsubscribe(ServerChannel session, String clientID, List<String> top
             //also will unsubscribe from the kernal
             try {
                 AndesMQTTBridge.getBridgeInstance().onSubscriberDisconnection(clientID,
-                        AndesMQTTBridge.SubscriptionEvent.UNSUBSCRIBE);
+                        AndesMQTTBridge.SubscriptionEvent.UN_SUBSCRIBE);
```

Comment:
:+1:  , 'unsusbcribe' sounds more readable, it was initially written as 'un-subscribe' due to it being detected as a typo :)